### ISSUE TITLE
[Feature Request] Allow syntax hint for yield in rules

### ISSUE THREAD

**Issue by gander** [laravel-idea/plugin#378]
Please add syntax suggestions in yield fields.
![1](https://user-images.githubusercontent.com/421597/131331527-2521e890-e81c-4330-820e-fc820d145b9a.png)
![2](https://user-images.githubusercontent.com/421597/131331530-316acf19-a2b2-4066-9e38-7cbad86750fc.png)

**Comment by gander** [comment]
Has anything changed since the last comment?

**Comment by gander** [comment]
☠️ is it alive?

**Comment by adelf** [comment]
Yes. There are some difficulties with it, but it might appear in one of the next releases.

**Comment by adelf** [comment]
Implemented in 11.5. Thank you.
